### PR TITLE
Quickfix: Ensure start epoch of LookupInEpochRequest and MonitoringRequest is in PAD

### PR DESCRIPTION
**Pull request by masomel**
Otherwise directory.LookupInEpoch() fails with ErrorDirectory, though a start epoch that is > the latest epoch indicates a malformed client message.

### CODE REVIEW

**Comment by arlolra** [comment]
Wouldn't hurt to add a test.

File: protocol/directory_test.go [hunk 1]
```diff
@@ -285,18 +285,56 @@ func TestDirectoryKeyLookupInEpoch(t *testing.T) {
...
 		Type:    MonitoringType,
-		Request: &MonitoringRequest{"alice", uint64(2), uint64(0)},
+		Request: &MonitoringRequest{"alice", uint64(6), uint64(10)},
 	}
 	_, err := d.HandleOps(req)
 	if err != ErrorMalformedClientMessage {
 		t.Fatal("Expect error", ErrorMalformedClientMessage, "got", err)
 	}
+
+	// Send an invalid MonitoringRequest (startEpoch < EndEpoch)
```
Comment (arlolra) [inline protocol/directory_test.go]: (StartEpoch > EndEpoch)

**Comment by c633** [comment]
LGTM.

**Comment by arlolra** [comment]
LGTM, too.

**Pull request closed without merging**